src/components/tree/phyloTree/grid.js

```diff
@@ -357,25 +368,29 @@ export const addTemporalSlice = function addTemporalSlice() {
 
   /* the gray region between the root (ish) and the minimum date */
   if (Math.abs(xWindow[0]-rootXPos) > minPxThreshold) { /* don't render anything less than this num of px */
-    this.groups.temporalWindow.append("rect")
-      .attr("x", rightHandTree ? xWindow[0] : 0)
-      .attr("width", rightHandTree ? totalWidth-xWindow[0]: xWindow[0])
-      .attr("y", 0)
+    const xEnd_start = rightHandTree ? totalWidth - xWindow[0] : xWindow[0]; // ending X coordinate of the "start" rectangle
```

Comment:
It's important to address the case of two-trees, where the x-axis is flipped. Currently (e.g.) localhost:4000/ncov/2020-02-11:ncov/2020-02-09?dmax=2020-01-26&m=num_date is broken as somewhere in this PR (probably not this line) we are assuming x-values are left-to-right